cli/util/api.go

```diff
@@ -214,6 +319,58 @@ func GetFullNodeAPIV1(ctx *cli.Context) (v1api.FullNode, jsonrpc.ClientCloser, e
 	return v1API, closer, nil
 }
 
+func GetFullNodeAPIV1New(ctx *cli.Context) (v1api.FullNode, jsonrpc.ClientCloser, error) {
```

Comment:
I think it's fine to call this `GetFullNodeAPI`, and the other one `GetFullNodeAPISingle`, and have everything go through `GetFullNodeAPI`, so everything gets multi-node support

Reply:
There's already something called GetFullNodeAPI which I believe points to the v0 API?
Changed the name to the default one

Reply:
Sorry meant GetFullNodeAPI**V1**